script.js

```diff
@@ -1,49 +1,47 @@
 // HTML Selectors
-const table = document.getElementById('mainTable')
-const btn = document.getElementById('btn')
-const dateField = document.getElementById('dateForCheckingAvailability')
-const districtsField = document.getElementById('districtCodes')
+const table = document.getElementById('mainTable');
+const btn = document.getElementById('btn');
+const dateField = document.getElementById('dateForCheckingAvailability');
+const districtsField = document.getElementById('districtCodes');
 
 // Init variables
-let dateOfAvailability = `${new Date().getDate() + 1}-${
-  new Date().getUTCMonth() + 1
-}-${new Date().getFullYear()}`
-dateField.value = dateOfAvailability
-let inputDistrictCodes = districtsField.value.replace(/\s/g, '').split(',')
-let isFilteredResult = false
+let dateOfAvailability = `${new Date().getDate() + 1}-${new Date().getUTCMonth() + 1}-${new Date().getFullYear()}`;
+dateField.value = dateOfAvailability;
+let inputDistrictCodes = districtsField.value.replace(/\s/g, '').split(',');
+let isFilteredResult = false;
 
 // District Lookup Table
 //TODO: Add more districts
 const districtMap = new Map([
   [392, 'Thane'],
   [395, 'Mumbai'],
-])
+]);
 
 const getVaccineAvailablity = async (districtIds, dateToCheck) => {
-  const parsedData = []
+  const parsedData = [];
   const options = {
     eighteen: document.getElementById('optionsAge18plus').checked,
     fortyFive: document.getElementById('optionsAge45plus').checked,
     doseOne: document.getElementById('optionsDose1').checked,
     doseTwo: document.getElementById('optionsDose2').checked,
-  }
+  };
 
   if (!options.eighteen && !options.fortyFive) {
-    alert('Please select which age(s) you are looking for.')
-    return
+    alert('Please select which age(s) you are looking for.');
```

Comment:
I am planning to keep the auto refresh feature in the code. So this alert might be too intrusive if we let it show up every time a search is made without any of the ages being selected.
To prevent this, I will have to setup a condition in main() for not to search at all if these are not checked. In doing so, this code block may never run. Just a heads up.

Reply:
Same with !dose1 && !dose2